app/assets/javascripts/Components/Modals/submission_file_upload_modal.jsx

```diff
@@ -57,6 +129,8 @@ class SubmissionFileUploadModal extends React.Component {
                 onChange={this.handleFileUpload}
               />
             </div>
+            <h3>{I18n.t("file_name")}</h3>
```

Comment:
Don't make this an `h3`. Instead, this should be a *label* for the input element (this is important for accessibility). Note that labels need to have an `htmlFor` element that matches the `id` element of the input (which can be a `select` as well).

Visually, the label should:

1. have the text `(optional) Rename file to`
2. appear beside, rather than above, the input element, like the other label/input pairs on the modal